projects/kubernetes-sigs/image-builder/buildspecs/build-1-21-ubuntu-ova.yml

```diff
@@ -14,12 +14,11 @@ env:
 
 phases:
   # On AL2, the Cargo build system requires the openssl-devel package
-  # for installing OpenSSL libraries and the pkgconfig utility to 
-  # locate these headers/libs
+  # for installing OpenSSL headers/libraries
   install:
     run-as: root
     commands:
-      - yum install -y openssl-devel pkgconfig
+      - yum install -y openssl-devel
```

Comment:
You want to just put this on the builder base?

Reply:
I did so [here](https://github.com/aws/eks-distro-build-tooling/commit/7a4552d57c9eb4bc35428ecf537bed15fb135c83) the other day, but the ensuing postsubmit failed because of the ARM build issues. Will remove these after a successful builder-base build.